Lib/swigfragments.swg

```diff
@@ -88,3 +88,11 @@
 %fragment("<memory>", "header") %{
 #include <memory>
 %}
+
+%fragment("<utility>", "header") %{
+#include <utility>
+%}
+
+%fragment("<cctype>", "header") %{
+#include <cctype>
+%}
```

Comment:
This new `<cctype>` fragment doesn't actually seem to be used (nor does `<cctype>` anywhere in the SWIG source in fact).